### PR TITLE
The Restaurant Page

### PR DESCRIPTION
**Pull request by vidhishweta01**
It is restaurant opening page has three pages about us, menu, contact us
built with javascript using webpack 
the live link is given below:
https://vidhishweta01.github.io/restaurant_page/

### CODE REVIEW

**Review by Forison — CHANGES_REQUESTED** [review]
## Status::Changes required❌ 

# 👋hello @vidhishweta01 

#### You have a done a good job on this one so far, Good work with the UI👏👏👏 

* [x] The contents of each tab are stored in a module. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))
- let us seperate the code in the index.js into different modules, we can have modules called about.js, menu.js contact.js navbar.js ... 

* [x] It features a tabbed menu. When a tab is clicked the content is replaced ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))
- Currently I have to reload the page to visit any of the tab, I am unable to go to a tab from another selected tab. ie I can't navigate to `menu` from `about us`.
For example When user selects about us, the content of the dom should be replaced by the content in the about.js modules

* [x] Let us use ES6 syntax in defining all our functions ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))

* [x] Let us state explicitly what to do to get the app set up and launched in the local environment. Let's include all commands needed.

Check comment below👇

**[ Optional ]** 

*  [ ]  It is better to always capitalize your commit messages

* [ ] Using bullet point to state the steps required to set up and run the application the local environment is more friendly to visitors.

#### That is it kindly make the changes and submit the project for a re-review🤗

#### Happy coding💪 

Addo Forison

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+  contact_us.setAttribute('class','contact-us w-25 px-0 py-2 btn-dark'); //eslint-disable-line
+  contact_us.innerHTML = _.join(['CONTACT US']); //eslint-disable-line
+
+  row.append(about_us, menu, contact_us); //eslint-disable-line
+  container_fluid.append(row); //eslint-disable-line
+  element.append(container_fluid); //eslint-disable-line
+  return element;
+}
+
+function component2() {
```
Comment (Forison) [inline src/index.js]: Let us arrow functions in our functions definitions here and all other places

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+  about_us.innerHTML = _.join(['ABOUT US']); //eslint-disable-line
+  const menu = document.createElement('button');
+  menu.setAttribute('class', 'menu w-25 px-0 py-2 btn-dark');
+  menu.innerHTML = _.join(['MENU']);
+  const contact_us = document.createElement('button'); //eslint-disable-line
+  contact_us.setAttribute('class','contact-us w-25 px-0 py-2 btn-dark'); //eslint-disable-line
+  contact_us.innerHTML = _.join(['CONTACT US']); //eslint-disable-line
+
+  row.append(about_us, menu, contact_us); //eslint-disable-line
+  container_fluid.append(row); //eslint-disable-line
```
Comment (Forison) [inline src/index.js]: * [ ] Do not make modfiy linter configs - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)))

**Review by MkrtichSargsyan — CHANGES_REQUESTED** [review]
## _Status:_ Changes Requested :recycle:

## Hello Shweta Srivastava


## To Highlight :tada:

1. Great work making the necessary corrections from the previous reviews.
2. You have a descriptive readme file.


However, there are some changes that you need to make in order to get your project approved

- [ ] Kindly write commit messages in simple present tense
- [x] Use ES6 syntax in defining all functions [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png)

- [x] Necessary run common commands like build, development should be in package.json inside `scripts'.screenshot
Here is a basic example

      "scripts": {
        "dev": "webpack --mode development",
        "build": "webpack --mode production",
        "start": "webpack-dev-server --mode development --open" // optional
      },

<hr>

Please, make the REQUIRED changes and submit your request again! :grinning:
Good luck. 

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

**Comment by vidhishweta01** [comment]
ES 6 syntax has been used throughout the project

**Comment by MkrtichSargsyan** [comment]
> ES 6 syntax has been used throughout the project

This is not an ES 6 syntax, please use arrow functions instead

function aboutUs() {
  const element = document.createElement('div');
  element.setAttribute('class', 'about');
  const h1 = document.createElement('h1');
  h1.innerHTML = _.join([' the best tasting experience']);
  const par = document.createElement('p');
  par.innerHTML = _.join(['Granny offers you']);
  const para = document.createElement('p');
  para.setAttribute('class', 'ml-0 mr-4');
  para.innerHTML = _.join(['Granny is restaurant, bar and cafe, we have awesome recipes and most talented chefs in town']);
  const button = document.createElement('h2');
  button.innerHTML = _.join(['GET A RESERVATION NOW']);
  const btn = document.createElement('button');
  btn.setAttribute('class', 'butto');
  btn.append(button);
  element.append(par, h1, para, btn);
  return element;
}

**Comment by vidhishweta01** [comment]
Is it depricated to use some traditional feature in the project??
https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript/lessons/restaurant-page

https://www.tutorialspoint.com/es6/es6_functions.htm
Arrow convention is causing error, this method is also given in E6 syntax 

Screenshot from 2021-06-27 23-35-42

Screenshot from 2021-06-27 23-42-03

**Comment by vidhishweta01** [comment]
Screenshot from 2021-06-27 23-43-26

All the functions referred are being called in different places throughout the code hence cannot be anonymous 
Arrow functions are lambda function please do refer the above screenshot and the link given below:
https://www.tutorialspoint.com/es6/es6_functions.htm

**Review by Georjane — APPROVED** [review]
## Status: PR Approved 💯 🥇 
### Hello team ✋🏽 

Congratulations on successfully completing this project. Your design is so beautiful!
Thank you for making the required changes from the previous Code Reviewer. You have met all requirements for this project to be approved. 
It is a thumbs up for me. 👍🏽 
You can go ahead and merge this PR.

Good luck on your next project and Happy Coding 💻

**Pull request merged**